airgun/browser.py

```diff
@@ -87,9 +87,9 @@ def get_browser(self):
         :raises: ValueError: If wrong ``provider`` or ``browser`` specified.
         """
         if self.provider == 'selenium':
-            return self._get_selenium_browser()
+            return self._get_selenium_browser(), None
         elif self.provider == 'remote':
-            return self._get_remote_browser()
+            return self._get_remote_browser(), self.web_kaifuku.webdriver_options.command_executor
```

Comment:
Quick question: How are we utilizing the command_executor?

Reply:
to find the grid hostname